### PR TITLE
Use defaults in fabrica config whenever possible

### PR DESCRIPTION
**Pull request by dzikowski**

### CODE REVIEW

File: e2e/__snapshots__/extendConfig.test.js.snap [hunk 1]
```diff
@@ -440,11 +443,6 @@ Object {
         "port": 7050,
       },
     ],
-    "organization": Object {
-      "domain": "root.com",
-      "mspName": "OrdererMSP",
-      "name": "Orderer",
-    },
```
Comment (Hejwo) [inline e2e/__snapshots__/extendConfig.test.js.snap:447]: hm... don't get it - why it's removed from extendConfig ?  
Shouldn't extendConfig show configuration after all defaults applied?

File: e2e/__snapshots__/extendConfig.test.js.snap [hunk 2]
```diff
@@ -1341,11 +1337,6 @@ Object {
...
```
Comment (Hejwo) [inline e2e/__snapshots__/extendConfig.test.js.snap:1348]: hm... don't get it - why it's removed from extendConfig ?
Shouldn't extendConfig show configuration after all defaults applied?

File: e2e/__snapshots__/extendConfig.test.js.snap [hunk 3]
```diff
@@ -1988,11 +1982,6 @@ Object {
...
```
Comment (Hejwo) [inline e2e/__snapshots__/extendConfig.test.js.snap:1995]: hm... don't get it - why it's removed from extendConfig ?
Shouldn't extendConfig show configuration after all defaults applied?

File: e2e/schemaFilesMatch.test.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+const commands = new TestCommands('./e2e/__tmp__/schema-files-match-tests', '../../..');
+
+describe('schema files match', () => {
+  beforeEach(() => commands.cleanupWorkdir());
+
+  beforeAll(() => {
+    process.env.FABRICA_CONFIG = '<absolute path>';
+    process.env.CHAINCODES_BASE_DIR = '<absolute path>';
+  });
```
Comment (Hejwo) [inline e2e/schemaFilesMatch.test.js:15]: :+1:

File: generators/extend-config/configTransformers.js [hunk 5]
```diff
@@ -95,38 +97,35 @@ function transformCaConfig(caJsonFormat, orgName, orgDomainJsonFormat, caExposeP
 }
 
 function transformRootOrgConfig(rootOrgJsonFormat) {
-  const { domain } = rootOrgJsonFormat.organization;
-  const orderersExtended = transformOrderersConfig(
-    rootOrgJsonFormat.orderer,
-    domain,
-  );
+  const { domain, name } = rootOrgJsonFormat.organization;
+  const mspName = rootOrgJsonFormat.organization.mspName || defaults.organization.mspName(name);
```
Comment (Hejwo) [inline generators/extend-config/configTransformers.js:101]: :heart:

File: e2e/__snapshots__/extendConfig.test.js.snap (hunk 1 above)
Comment (dzikowski) [reply e2e/__snapshots__/extendConfig.test.js.snap:447]: it was a duplicate, see lines 425-427. I removed `organization` to keep it consistent with rootOrg

**Review by Hejwo — APPROVED** [review]

**Pull request merged**